python/graphstorm/trainer/ep_trainer.py

```diff
@@ -33,6 +33,15 @@
 class GSgnnEdgePredictionTrainer(GSgnnTrainer):
     """ Edge prediction trainer.
 
+    This class is used to train models for edge prediction tasks,
+    such as edge classification and edge regression.
+
+    It makes use of the functions provided by `GSgnnTrainer`
+    to define two main functions: `fit` that performs the training
+    for the model that is provided when the object is created,
+    and `eval` that evaluates a provided model against test and
+    validation data.
+
```

Comment:
Could you add an example of how to construct an Trainer instance, and call its fit() method.
You can refer to GS' own use of trainer as an example and simply it into Pseudo code.
You can refer to this PR for the format: https://github.com/awslabs/graphstorm/pull/456/files